mbuild/formats/charmm_writer.py

```diff
@@ -1049,6 +1049,67 @@ def _lengths_angles_to_vectors(lengths, angles, precision=6):
     return box_vectors
 
 
+def _check_fixed_bonds_angles_lists(
+    fixed_residues_bonds_angles_list,
+    fixed_bonds_angles_residues_variable_name,
+    residues,
+):
+    """Check the GOMC fixed bonds and angles lists for input errors.
+
+    Parameters
+    ----------
+    fixed_residues_bonds_angles_list : list of strings, [str, ..., str]
+        A list of the residues (i.e., molecules since GOMC currently considers a
+        a whole molecule as a residue) to have their bonds and/or angles held
+        rigid/fixed for the GOMC simulation engine.
+        In GOMC, the residues currently the molecules.
```

Comment:
```suggestion
        In GOMC, the residues currently the molecules.
```

Still not sure what this sentence is supposed to mean? Or I think this line can be removed, since you updated the description above it.

Reply:
I changed the statement above to :

In GOMC, the residues currently are the same for every bead or atom in the molecules. Therefore, when the residue is selected, the whole molecule is selected.